### DISCUSSION TITLE
New helper to check if some string is json

### DISCUSSION THREAD

**Discussion by Zen0x7** [laravel/framework#38623] [Ideas] [answered]
> Yeah. I'm pretty sure that should be a feature of PHP but no function is created. 

I have been looking for different implementations and the following is really nice:

```php
function is_json(string $var) {
    return json_decode($var) && json_last_error() === JSON_ERROR_NONE;
}
```

What you think?

**Comment by crynobone** [comment, marked as answer]
`json_validate` will be available in PHP 8.3: https://wiki.php.net/rfc/json_validate and you can already use polyfill from https://github.com/symfony/polyfill-php83 right now

**Reply by Zen0x7** [reply]
Thanks @crynobone for the answer. Apparently i have nice skill to see the future 🤣.
